src/main/java/uk/gov/companieshouse/chd/order/api/dto/MissingImageDeliveriesDTO.java

```diff
@@ -88,6 +92,14 @@ public void setPaymentReference(String paymentReference) {
 		this.paymentReference = paymentReference;
 	}
 
+	public String getFilingHistoryType() {
+		return filingHistoryType;
+	}
+
+	public void setFilingHistoryType(String filingHistoryType) {
+		this.filingHistoryType = filingHistoryType;
+	}
+
```

Comment:
This shouldn't have tabs in it, it should use spaces instead. If the entire class is like this, we should raise the formatting as a separate Jira issue. If not, this should be re-formatted.